### ISSUE TITLE
Idea: extra functionality for Hidden rewards helper.

### ISSUE THREAD

**Issue by belikoviv** [dsiekiera/foe-helfer-extension#837]
Incidents sometime placed on roads behind tall buildings that make its completely unvisible and unfindable.
Is it possible to add eye icon in incident rows in Hidden Rewards window and show incident place on city map (as in Production overview)

**Comment by tartetatin18** [comment]
yes, that would be very helpul

**Comment by scibuff** [comment]
Agreed! This would be great!

**Comment by andrgin** [comment]
Unfortunately we do not have the information where inc8dents are located

**Comment by Jay1181** [comment]
Foe Helper has been showing incidents on "small roads" and "large roads" since I installed. I have only *one* road throughout my city. Also, I cannot find the incidents. Right now, FH shows that I should have 3 incidents in the roads. I have magnified my city, and picked up *every* building, and have found *zero* incidents. This has been going on since I installed FH.
 FH is a *FANTASTIC* tool, but this part is frustrating. I am guessing that I don't really have incidents in the roads, but FH is getting a false reading. And I don't understand it saying that I have two different kinds of roads.
